core/src/test/scala/org/apache/spark/sql/delta/UpdateScalaSuite.scala

```diff
@@ -60,6 +59,23 @@ class UpdateScalaSuite extends UpdateSuiteBase  with DeltaSQLCommandTest {
       Row(100, 101) :: Row(100, 101) :: Row(3, 30) :: Row(4, 40) :: Nil)
   }
 
+  test("update usage test - with space in path") {
+    val tmpDirForTest = System.getProperty("java.io.tmpdir").concat("/test with space/")
```

Comment:
We have an existing similar test in https://github.com/delta-io/delta/blob/eab445a15f478ed2454cf152edb795e9fece91f6/core/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala#L919 . Could you move this test there? You can take a look at how we can a test directory.